post-rs-fish/v1/scripts/python/post-rs-fish.py

```diff
@@ -0,0 +1,92 @@
+import argparse
+import numpy as np
+
+import io_utils.zarr_utils as zarr_utils
+
+
+def _floattuple(arg):
+    if arg is not None and arg.strip():
+        return tuple([float(d) for d in arg.split(',')])
+    else:
+        return ()
+
+
+def _define_args():
+    args_parser = argparse.ArgumentParser()
+    args_parser.add_argument('-i', '--input',
+                             dest='input',
+                             type=str,
+                             required=True,
+                             help = "spots input file using voxel coordinates")
+
+    args_parser.add_argument('-o','--output',
+                             dest='output',
+                             type=str,
+                             required=True,
+                             help = "output file in real coordinates")
+
+    args_parser.add_argument('--image-container',
+                             dest='image_container',
+                             type=str,
+                             help = "image container")
+    args_parser.add_argument('--image-subpath', '--image-dataset',
+                             dest='image_dataset',
+                             type=str,
+                             help = "image subpath")
+
+    args_parser.add_argument('--voxel-spacing', '--voxel_spacing',
+                             dest='voxel_spacing',
+                             type=_floattuple,
+                             help = "voxel spacing")
+    args_parser.add_argument('--expansion-factor', '--expansion_factor',
+                             dest='expansion_factor',
+                             type=float,
+                             default=0.,
+                             help='Sample expansion factor')
+
+
+
+    return args_parser
+
+
+def _post_process_rsfish_csv_results(args):
```

Comment:
I assume this logic was adapted from https://github.com/JaneliaSciComp/multifish/blob/master/containers/spot_extraction/scripts/python/post_rsfish.py

I think it's better to move things as-is, and then refactor them in a second step. That way you have a full history of what was working and how it changed.